operations/_scripts/deploy/deploy.sh

```diff
@@ -30,6 +30,14 @@ if [[ "$CREATE_DOMAIN" == "true" ]]; then
   /bin/bash $GITHUB_ACTION_PATH/operations/_scripts/generate/generate_acm_tf.sh
 fi
 
+# Generate Ansible
```

Comment:
Is "Generate Ansible" actual code comment for this?
Sounds like it's more about using the Ansible vars from the user's repo.